types/node/v20/package.json

```diff
@@ -1,15 +1,15 @@
 {
     "private": true,
     "name": "@types/node",
-    "version": "20.14.9999",
+    "version": "20.15.9999",
     "nonNpm": "conflict",
     "nonNpmDescription": "Node.js",
     "projects": [
         "https://nodejs.org/"
     ],
     "tsconfigs": ["tsconfig.dom.json", "tsconfig.non-dom.json"],
     "dependencies": {
-        "undici-types": "~5.26.4"
+        "undici-types": "~6.13.0"
```

Comment:
Did this actually get updated in 20.15, or is this correcting a mistake? I don't see this upgrade in the changelog (though 20.16 does get 6.19)

Reply:
It was actually updated earlier, in [v20.13.0](https://github.com/nodejs/node/releases/tag/v20.13.0)

Reply:
PR: https://github.com/nodejs/node/pull/52493